MVP Module/Tests/Feed UI/FeedUISnapshotTests.swift

```diff
@@ -14,23 +14,23 @@ class FeedUISnapshotTests: XCTestCase {
 	//
 	//  ***********************
 
-//	func test_emptyFeed() {
-//		let sut = makeSUT()
-//
-//		sut.display(emptyFeed())
-//
-//		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light)), named: "EMPTY_FEED_light")
-//		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark)), named: "EMPTY_FEED_dark")
-//	}
-//
-//	func test_feedWithError() {
-//		let sut = makeSUT()
-//
-//		sut.display(errorMessage: "An error message")
+	func test_emptyFeed() {
+		let sut = makeSUT()
+
+		sut.display(emptyFeed())
+
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light)), named: "EMPTY_FEED_light")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark)), named: "EMPTY_FEED_dark")
+	}
+
 //
-//		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light)), named: "FEED_WITH_ERROR_light")
-//		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark)), named: "FEED_WITH_ERROR_dark")
-//	}
+	func test_feedWithError() {
+		let sut = makeSUT()
+		sut.display(FeedErrorViewModel(errorMessage: "An error message"))
```

Comment:
Following from my previous comment, with a helper method you can decouple your test from the implementation details of using a `FeedErrorViewModel`. That way your tests won't break if you move away from MVVM for instance.
```suggestion
		sut.display(errorMessage: "An error message")
```

Reply:
done